test/crud-with-options.test.js

```diff
@@ -354,9 +354,81 @@ describe('crud-with-options', function () {
   });
 
   describe('deleteById', function() {
-    it('should allow deleteById(id)', function () {
-      User.deleteById(1);
+
+    beforeEach(seed);
+
+    it('should allow deleteById(id) - success', function (done) {
+      User.findOne(function (e, u) {
+        User.deleteById(u.id, function(err, deleted) {
+          should.not.exist(err);
```

Comment:
Please use `if (err) return done(err)`, it gives more friendly test failure messages.